circuits/offchain/credentialAtomicQueryV3OffChain.circom

```diff
@@ -2,11 +2,13 @@ pragma circom 2.1.5; /* chage to 2.1.6 */
 
 include "./credentialAtomicQueryMTPOffChain.circom";
 include "./credentialAtomicQuerySigOffChain.circom";
+include "../lib/linked/linkId.circom";
 
 template credentialAtomicQueryV3OffChain(issuerLevels, claimLevels, valueArraySize) {
     // common outputs for Sig and MTP
     signal output merklized;
     signal output userID;
+    signal output linkID;
```

Comment:
Please move it both linkID and linkNonce to the end of signals block (after issuerAuthState)